### ISSUE TITLE
Trouble getting akka remoting to start

### ISSUE THREAD

**Issue by dwschulze** [jasongoodwin/learning-akka#11]
I'm trying to run your code without activator.  It works if I run it with `sbt publishLocal` and `sbt run` as downloaded.  When I copy your build.sbt and src/ directory to another project akka remoting doesn't start.  I've also added the activator and activator-launch-1.2.10.jar file but that makes no difference.

What else do I need to get this to work without activator?

**Comment by jasongoodwin** [comment]
Yeah sbt instead of activator should be fine. I regret using activator -
didn't expect it to dissappear!
